### PR TITLE
Add wmflabs.org to MDFP

### LINKED ISSUE #1178: en.wikipedia.org breaks page
https://tools.wmflabs.org/pageviews/?project=en.wikipedia.org&platform=all-access&agent=user&range=latest-20&pages=

`en.wikipedia.org` breaks page.

### PR DESCRIPTION
**Pull request by cowlicks**
Fixes #1178.

### CODE REVIEW

**Comment by ghostwords** [comment]
Edited description to link to the issue this helps resolve.

**Comment by ghostwords** [comment]
Follows up on  #1525.

**Comment by ghostwords** [comment]
Example sites: https://github.com/EFForg/privacybadger/pull/1525#discussion_r129689462

File: src/js/multiDomainFirstParties.js [hunk 1]
```diff
@@ -182,7 +169,7 @@ var _multiDomainFirstPartiesArray = [
   ["walmart.com", "wal.co"],
   ["wellsfargo.com", "wf.com"],
   ["wikia.com", "wikia.net", "nocookie.net"],
-  ["wikipedia.org", "wikimedia.org", "wikimediafoundation.org", "wiktionary.org",
+  ["wmflabs.org","wikipedia.org", "wikimedia.org", "wikimediafoundation.org", "wiktionary.org",
```
Comment (ghostwords) [inline src/js/multiDomainFirstParties.js]: Nit: missing a separating space.

Could we move it to the back of Wikipedia entries somewhere? We should have the most recognizable sites up front probably.

**Comment by ghostwords** [comment]
What do you think about adding a unit test that catches this scenario? Iterate over all MDFP entries, fail for the ones where `getBaseDomain(entry) != getBaseDomain("subdomain." + entry)`. A linter for our MDFP list.

**Comment by cowlicks** [comment]
Closing in favor of #1568

**Comment by cowlicks** [comment]
@ghostwords in #1567 I started adding tests for what we expected data in the mdfp list to look like ([link](https://github.com/EFForg/privacybadger/pull/1567/files#diff-3ee622e77096162c22e4e9bbc82e1d78)). I'd eventually like tests there to enforce our expectations of things like the mdfp, yellowlist, dnt hashes, and anyother new trusted data that gets added. The enforcement would be based on the rules from #1515.

**Comment by ghostwords** [comment]
Could you add MDFP linting (so that we never run into this issue again) to #1568 (the PR that fixes this issue)?

**Comment by cowlicks** [comment]
@ghostwords I'd prefer to add it in a seperate PR.

**Comment by ghostwords** [comment]
MDFP linting was added in PR #1591.

**Pull request closed without merging**